chart/epinio-installer/templates/NOTES.txt

```diff
@@ -6,8 +6,9 @@ You can get the IP address of your Ingress controller by passing this command:
 Make sure your domain points to the IP address of your Ingress controller (1.2.3.4).
 {{ end }}
 To interact with your Epinio installation, you have to download the latest epinio binary https://github.com/epinio/epinio/releases.
-
-Update the api location and credentials with:
+In this case, update the api location and credentials with:
 `epinio config update`
 
+Otherwise, if you install Epinio through Rancher (or imported your Cluster in Rancher), you can control Epinio through the Epinio menu you will find in Global Apps.
```

Comment:
```suggestion
Otherwise, if you installed Epinio with Rancher (or imported your Cluster in Rancher), you can control Epinio through the Epinio menu you will find in Global Apps.
```